.github/workflows/build-signed-deb.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       with:
         name: LinuxInstallers
 
-    - uses: Azure/login@v1.1
+    - uses: azure/login@v1
```

Comment:
This is for https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/